src/fragmentarium/ui/fragment/TeiExport.js

```diff
@@ -0,0 +1,138 @@
+export function teiExport(fragment) {
```

Comment:
Make a unit test for the XML conversion.

Reply:
will do when all other issues are resolved (and the XML output is as desired)